src/main/java/io/tja/osrs/shattered_relics_fragment_presets/ShatteredRelicsFragmentPresetsConfig.java

```diff
@@ -68,4 +68,15 @@ default boolean shiftClickToggleFragment() {
     default boolean showExtraButtons() {
         return false;
     }
+
+    @ConfigItem(
```

Comment:
Open to feedback on the config for this! Being able to configure the page size felt like a good feature and 10 felt like a good default since that's about the amount that fits on the screen in fixed mode.

I know you mentioned some concern about disrupting users. I'm hoping this implementation is unobtrusive enough that it's okay to turn on by default. Otherwise, I think there are a couple options:

- set the default page size to a high enough value so that by default all presets fit on a single page anyway, meaning users have to change the page size to get the paginated experience
- OR keep a more reasonable default page size like 10, but add a separate, off by default, option to control whether pagination happens at all.

Let me know what you think!

Reply:
this is really cool! love how it works live while you're using the plugin. I'll have to think a bit about how I think would be best to keep it from being too disruptive, but I think this might be minimal enough that it's worth it. I'll play around a bit